models/zeroex/polygon/zeroex_polygon_nft_fills.sql

```diff
@@ -58,7 +58,7 @@ WITH tbl_cte_transaction AS
             , erc1155FillAmount  as nft_cnt
             , CASE
                 WHEN erc20Token = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'
-                THEN '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619'
+                THEN '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270'
```

Comment:
can we use this logic to cover all the bases?
**_erc20Token IN ('0x0000000000000000000000000000000000001010','0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') THEN '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270'_**

Reply:
good call - all set with this change

Reply:
yes. this is perfect. thank you @RantumBits